pom.xml

```diff
@@ -99,7 +99,7 @@
         <mockito_version>2.23.4</mockito_version>
         <!-- Build args -->
         <argline>-server -Xms256m -Xmx512m -XX:PermSize=64m -XX:MaxPermSize=128m -Dfile.encoding=UTF-8
```

Comment:
Please remove `-XX:PermSize=64m -XX:MaxPermSize=128m`, there is no need to keep it.